### ISSUE TITLE
Disable 2FA not working on Adobe Commerce 2.4.6-p2

### ISSUE THREAD

**Issue by Mr-Leonerrr** [rewardenv/reward#54]
### Description
Lately, I've been getting this error when trying to install Magento on version 2.4.6-p2 by disabling 2FA. This was working fine in previous versions, is there something that has changed, or should I run another command?

```shell
ERROR   error running bootstrap command: error bootstrapping magento2: error during magento 2 installation: cannot run bin/magento module:disable Magento_TwoFactorAuth: error running command: docker-compose: exit status 1
```
I usually do
```shell
reward bootstrap --disable-tfa --magento-mode developer --magento-type enterprise --with-sampledata --magento-version 2.4.6-p2
```

### System Information
INFO    reward version: 0.4.8
INFO    GOOS: linux
INFO    GOARCH: amd64

**Comment by janosmiko** [comment]
This looks promising for Magento 2.4.6+.
Now, I'm on holiday but I'll add it to reward when I'm back.

https://magento.stackexchange.com/a/365904

**Comment by janosmiko** [comment]
It should be fixed in v0.4.9

**Comment by janosmiko** [comment]
I had to reopen this issue, because the library I'm using is not supporting patch versions during version comparison and it reports v2.4.6-p2 < v2.4.6.

I changed the minimum version for this comparison to v2.4.5.99.

A new release is building right now (v0.4.10). That should work when you want to bootstrap a Magento 2.4.6-p2. (If you want to use a previous project, you'll have to change the PHP_VERSION to 8.2).
